lib/middleman-hashicorp/partials/_mega.html.erb

```diff
@@ -33,7 +35,7 @@
                   <p class="mega-nav-h2">Any infrastructure for any&nbsp;application</p>
                 </div>
                 <div class="mega-nav-body-header-item">
-                  <a href="https://www.hashicorp.com/devops-defined/" class="mega-nav-btn"><img src="<%= image_path("mega-nav/logo-hashicorp.svg") %>" alt="HashiCorp Logo" /> Learn the HashiCorp Suite <svg class="mega-nav-icon mega-nav-icon-angle-right" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#mega-nav-icon-angle"></use></svg></a>
```

Comment:
Two things here:

1. Was the change to the URL intentional?
1. There's an `&nbsp;` in here instead of a regular space - is that intended?